docs/feature_midi.md

```diff
@@ -0,0 +1,246 @@
+|`MI_TRNS_1` |         |Set transposition to +1 semitone |
+|`MI_TRNS_2` |         |Set transposition to +2 semitones|
+|`MI_TRNS_3` |         |Set transposition to +3 semitones|
+|`MI_TRNS_4` |         |Set transposition to +4 semitones|
+|`MI_TRNS_5` |         |Set transposition to +5 semitones|
+|`MI_TRNS_6` |         |Set transposition to +6 semitones|
+|`MI_TRNSD`  |         |Decrease transposition           |
+|`MI_TRNSU`  |         |Increase transposition           |
+|`MI_VEL_0`  |         |Set velocity to 0                |
+|`MI_VEL_1`  |         |Set velocity to 12               |
+|`MI_VEL_2`  |         |Set velocity to 25               |
+|`MI_VEL_3`  |         |Set velocity to 38               |
+|`MI_VEL_4`  |         |Set velocity to 51               |
+|`MI_VEL_5`  |         |Set velocity to 64               |
+|`MI_VEL_6`  |         |Set velocity to 76               |
+|`MI_VEL_7`  |         |Set velocity to 89               |
+|`MI_VEL_8`  |         |Set velocity to 102              |
+|`MI_VEL_9`  |         |Set velocity to 114              |
+|`MI_VEL_10` |         |Set velocity to 127              |
+|`MI_VELD`   |         |Decrease velocity                |
+|`MI_VELU`   |         |Increase velocity                |
+|`MI_CH1`    |         |Set channel to 1                 |
+|`MI_CH2`    |         |Set channel to 2                 |
+|`MI_CH3`    |         |Set channel to 3                 |
+|`MI_CH4`    |         |Set channel to 4                 |
+|`MI_CH5`    |         |Set channel to 5                 |
+|`MI_CH6`    |         |Set channel to 6                 |
+|`MI_CH7`    |         |Set channel to 7                 |
+|`MI_CH8`    |         |Set channel to 8                 |
+|`MI_CH9`    |         |Set channel to 9                 |
+|`MI_CH10`   |         |Set channel to 10                |
+|`MI_CH11`   |         |Set channel to 11                |
+|`MI_CH12`   |         |Set channel to 12                |
+|`MI_CH13`   |         |Set channel to 13                |
+|`MI_CH14`   |         |Set channel to 14                |
+|`MI_CH15`   |         |Set channel to 15                |
+|`MI_CH16`   |         |Set channel to 16                |
+|`MI_CHD`    |         |Decrease channel                 |
+|`MI_CHU`    |         |Increase channel                 |
+|`MI_ALLOFF` |         |Stop all notes                   |
+|`MI_SUS`    |         |Sustain                          |
+|`MI_PORT`   |         |Portmento                        |
+|`MI_SOST`   |         |Sostenuto                        |
+|`MI_SOFT`   |         |Soft Pedal                       |
+|`MI_LEG`    |         |Legato                           |
+|`MI_MOD`    |         |Modulation                       |
+|`MI_MODSD`  |         |Decrease modulation speed        |
+|`MI_MODSU`  |         |Increase modulation speed        |
+|`MI_BENDD`  |         |Bend pitch down                  |
+|`MI_BENDU`  |         |Bend pitch up                    |
```

Comment:
```suggestion
|`MI_BENDU`  |         |Bend pitch up                    |

### Configuration

Certain values are stored in the `midi_config` struct. This configuration is not persisted to EEPROM. By default, these values are:

|Configuration      |Value|Comments                 |
|-------------------|-----|-------------------------|
|Octave             |`4`  |Corresponds to `MI_OCT_2`|
|Transposition      |`0`  |                         |
|Velocity           |`127`|                         |
|Channel            |`0`  |                         |
|Modulation Interval|`8`  |                         |

For the above, the `MI_C` keycode will produce a C3 (note number 48), and so on.
```

Not quite sure on this, but maybe the descriptions for the note keycodes should be changed to eg. `C octave + 0` to further reinforce that they are relative to the current octave in the MIDI config. What do you think?

Reply:
I think something like that might be good, maybe give it some sort of name to make it easier to reference in the description and we can link it.